linux-ubuntu-musl-2404.pkr.js/linux-ubuntu-musl-2404.Dockerfile

```diff
@@ -0,0 +1,133 @@
+     fi
+
+# Build and install Clang, LLD, LLVM tools and runtimes
+WORKDIR /sources/llvm-project
+RUN cmake -G Ninja -S llvm -B build/musl \
+  -DLLVM_ENABLE_PROJECTS="clang;lld" \
+  -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi;libunwind;compiler-rt" \
+  -DLLVM_DEFAULT_TARGET_TRIPLE="${TARGET_TRIPLE}" \
+  -DLLVM_RUNTIME_TARGETS="${TARGET_TRIPLE}" \
+  -DLLVM_BUILTIN_TARGETS="${TARGET_TRIPLE}" \
+  -DLLVM_TARGETS_TO_BUILD="${LLVM_ARCH}" \
+  -DLLVM_ENABLE_EH=ON \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DCLANG_DEFAULT_RTLIB=compiler-rt \
+  -DCLANG_DEFAULT_LINKER=lld \
+  -DCLANG_DEFAULT_CXX_STDLIB=libc++ \
+  -DCLANG_DEFAULT_UNWINDLIB=libunwind \
+  -DRUNTIMES_${TARGET_TRIPLE}_COMPILER_RT_CXX_LIBRARY=libcxx \
+  -DRUNTIMES_${TARGET_TRIPLE}_COMPILER_RT_USE_LLVM_UNWINDER=ON \
+  -DRUNTIMES_${TARGET_TRIPLE}_COMPILER_RT_BUILD_LIBFUZZER=OFF \
+  -DRUNTIMES_${TARGET_TRIPLE}_COMPILER_RT_BUILD_ORC=OFF \
+  -DRUNTIMES_${TARGET_TRIPLE}_COMPILER_RT_BUILD_XRAY=OFF \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXX_ENABLE_SHARED=OFF \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXX_HAS_MUSL_LIBC=ON \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXX_STATICALLY_LINK_ABI_IN_STATIC_LIBRARY=ON \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXXABI_ENABLE_SHARED=OFF \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXXABI_USE_COMPILER_RT=ON \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXXABI_USE_LLVM_UNWINDER=ON \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBUNWIND_ENABLE_SHARED=OFF \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXX_ADDITIONAL_COMPILE_FLAGS="-I${SYSROOT_PATH}/include;-I${SYSROOT_PATH}/include/${TARGET_TRIPLE}" \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBCXXABI_ADDITIONAL_COMPILE_FLAGS="-I${SYSROOT_PATH}/include;-I${SYSROOT_PATH}/include/${TARGET_TRIPLE}" \
+  -DRUNTIMES_${TARGET_TRIPLE}_LIBUNWIND_ADDITIONAL_COMPILE_FLAGS="-I${SYSROOT_PATH}/include;-I${SYSROOT_PATH}/include/${TARGET_TRIPLE}" \
+  -DCMAKE_INSTALL_PREFIX="${SYSROOT_PATH}" \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  -DCMAKE_C_COMPILER=clang \
+  -DCMAKE_CXX_COMPILER=clang++ \
+  && ninja -C build/musl \
+    install-clang-stripped \
+    install-clang-resource-headers-stripped \
+    install-lld-stripped \
+    install-llvm-ar-stripped \
+    install-llvm-ranlib-stripped \
+    install-llvm-as-stripped \
+    install-llvm-nm-stripped \
+    install-llvm-objcopy-stripped \
+    install-llvm-strip-stripped \
+    install-builtins-stripped \
+    install-runtimes-stripped \
+  && echo "--sysroot=<CFGDIR>/.." > "${SYSROOT_PATH}/bin/${TARGET_TRIPLE}.cfg"
+
+# Create an empty libatomic archive to improve compatibility with builds requiring it on Linux
+# even though the functionality is already provided by compiler-rt.
+RUN ${SYSROOT_PATH}/bin/llvm-ar -r ${SYSROOT_PATH}/lib/libatomic.a
+
+FROM ${UBUNTU_24_04}
+
+ARG SYSROOT_PATH
+ARG DEBIAN_FRONTEND
+
+# Copy the sysroot from the previous stage
+COPY --from=build "${SYSROOT_PATH}" "${SYSROOT_PATH}"
+
+# Copy the just-built tipi release from previous CI stage
+ENV TIPI_DISTRO_MODE=all
```

Comment:
This line should be removed, using the `all` mode installs our distro provided clang13 which is not wanted for this kind of environments that provides the toolchain (cmake takes some tools from there like ranlib, objdump, dlltool, addr2line)

It would be great to make the missing tools come from the fresh clang musl build.